cli/app/app.go

```diff
@@ -1260,6 +1264,8 @@ func Up(c *cli.Context) {
 		dcCreated, err := client.Deployments(api.NamespaceDefault).Create(dc)
 		if err != nil {
 			logrus.Fatalf("Failed to create deployment controller %s: ", k, err)
+		} else {
+			fmt.Println("Deployment controller " + k + " has been created.")
```

Comment:
Consider using "%q"
```go
fmt.Printf("Deployment controller %q has been created.\n", k)
```
gives you something like `Deployment controller "web" has been created.`